frappe_types/frappe_types/type_generator.py

```diff
@@ -1,10 +1,25 @@
 import frappe
 from pathlib import Path
 from .utils import create_file
+import subprocess
+
 
 def create_type_definition_file(doc, method=None):
-    print("DocType created")
-    print(doc)
+
+    # Fetch Type Generation Settings Document
+    type_generation_settings = frappe.get_doc(
```

Comment:
We should not fetch the settings at the very beginning. If the cases below are not satisfied (dev mode off, migration etc), this will be a useless call to the database. 

Fetch the settings after we are sure that we need to generate the types and the app path exists.